synapse/storage/database.py

```diff
@@ -2060,68 +2060,20 @@ def make_in_list_sql_clause(
 
 
 def make_tuple_comparison_clause(
-    database_engine: BaseDatabaseEngine, keys: List[Tuple[str, KV]]
+    _database_engine: BaseDatabaseEngine, keys: List[Tuple[str, KV]]
```

Comment:
Can we remove this arg from all the call locations? Given we need to check that nothing uses kwargs, I don't see much point in keeping it around

Reply:
if you prefer, sure.